modules/highgui/src/window_w32.cpp

```diff
@@ -1874,21 +1874,41 @@ static void showSaveDialog(CvWindow* window)
     ofn.lStructSize = sizeof(ofn);
 #endif
     ofn.hwndOwner = window->hwnd;
+#ifdef HAVE_PNG   
     ofn.lpstrFilter = "Portable Network Graphics files (*.png)\0*.png\0"
+#else
+    ofn.lpstrFilter = "Windows bitmap (*.bmp;*.dib)\0*.bmp;*.dib\0"
+#endif
+#ifdef HAVE_JPEG
                       "JPEG files (*.jpeg;*.jpg;*.jpe)\0*.jpeg;*.jpg;*.jpe\0"
+#endif
+#ifdef HAVE_PNG 
                       "Windows bitmap (*.bmp;*.dib)\0*.bmp;*.dib\0"
```

Comment:
OK. thanks. i have another question to be sure . is "RadianceHDR(.hdr;.pic)" alway supported.